### ISSUE TITLE
Failed to Get plugin Dashboard

### ISSUE THREAD

**Issue by balajis2k** [grafana/grafana-zabbix#1611]
I installed my grafana in the zabbix server. I tried to connect to the dashboard but I'm facing "**Failed to Plugin Dashboard**".
Zabbix version : 6.4
Grafana version: 9.4
![image](https://user-images.githubusercontent.com/129855140/229790634-7673c316-f9cb-4f3e-88fc-c3bf222060e8.png)

Log:
logger=context userId=1 orgId=1 uname=admin t=2023-04-04T12:23:38.507595059Z level=error msg="Failed to get plugin dashboards" error="file does not exist" remote_addr=*.*.*.* traceID=
logger=context userId=1 orgId=1 uname=admin t=2023-04-04T12:23:38.507695462Z level=error msg="Request Completed" method=GET path=/api/plugins/alexanderzobnin-zabbix-datasource/dashboards status=500 remote_addr=*.*.*.* time_ms=21 duration=21.338693ms size=58 referer=http://localhost:3000/datasources/edit/uiyHxVLVz/dashboards handler=/api/plugins/:pluginId/dashboards/
I tried to follow [this](https://github.com/alexanderzobnin/grafana-zabbix/issues/1358#issuecomment-995309565), but after that my grafana server is not starting anymore.

**Comment by Tronyxsystem** [comment]
I've exactly the same issue on my new installation of grafana/zabbix =/

**Comment by wanit2** [comment]
Did you check Save$Test button in the Settings tab?

**Comment by balajis2k** [comment]
> Did you check Save$Test button in the Settings tab?

yeah, it got tested and it displayed my zabbix version, when i tried to select the dashboard option. I was getting failed to plugin dashboard error

**Comment by warderus** [comment]
yeah, similar problem

**Comment by Tronyxsystem** [comment]
<img width="1506" alt="Screenshot 2023-04-12 at 18 01 16" src="https://user-images.githubusercontent.com/33706692/231515831-b9aa7ea4-6f8d-4c7f-8681-d7be5c545e92.png">

Same issue.

New installation of zabbix in 6.4 and grafana in last version 9.4.7

**Comment by Marauderrrr** [comment]
It looks same with
https://github.com/alexanderzobnin/grafana-zabbix/issues/1407

**Comment by moni92gm** [comment]
I find myself in the same situation. I wanted to use the plugin for a university project but if there isn't a solution I won't be able to continue with the development. Could someone tell me if the synchronization works even if the dashboards are not there and if one could be created from scratch? I've been trying for days to find a solution.

**Comment by ted678** [comment]
Same issue
Grafana: 9.4.3
alexanderzobnin-zabbix-app: 4.3.1
![image](https://user-images.githubusercontent.com/35244034/234483378-83f04e85-f937-4a9c-bc94-ddf5651f28a0.png)
![image](https://user-images.githubusercontent.com/35244034/234483801-88ccc967-e31c-4264-a742-1d0f89fb8c94.png)
Log:
![image](https://user-images.githubusercontent.com/35244034/234483656-c475d6d9-f97f-48d7-91c7-a62c9a79da69.png)

**Comment by wanit2** [comment]
Thanks for help, ted678! Unfortunately it's not actually for me now, anyway thank you very much one more time!

**Comment by Tronyxsystem** [comment]
No news ? Solution ?

**Comment by balajis2k** [comment]
> No news ? Solution ?

No, No reply or solution. Still having issue to connect Grafana with zabbix

**Comment by githubzjb** [comment]
Guys, downgrading the Zabbix plugin version to 4.2.10 solves the problem

**Comment by Tronyxsystem** [comment]
> 

In 4.2.10 we have an other issue with the var user / username as you can read here => https://github.com/grafana/grafana-zabbix/issues/1583

And the fix doesn't work ...

**Comment by aaujon** [comment]
Hi,
Same for me in 4.4.0, a workaround it to copy dashboards/ directory from git repository to local datasource-zabbix/ directory. Looks like a packaging issue.

**Comment by alexanderzobnin** [comment]
Ah, seems, dashboard folder is not copied during the build process. I'll fix it.

**Comment by h0102** [comment]
我也碰到了同样的问题，zabbix6.4.3和GrafanaV10.1.1,加载的是V4.4.1的数据源插件，也是提示
![image](https://github.com/grafana/grafana-zabbix/assets/144880274/4e29117c-dca4-4502-b336-3b06444cbaf1)
